src/theme/components/customTable.ts

```diff
@@ -42,40 +86,40 @@ export const baseTRStyles = (
 export const baseCellStyles = (
   columnHeadersBackgroundColor = "",
   columnHeadersTextColor = "",
-  showRowDividers = false,
-  useRowHeaders = false
+  tableTextSize = "",
```

Comment:
Looks like tableTextSize is optional, can this be omitted?

Reply:
`tableTextSizeI` is being used on line 97, so it is needed here.  It is optional on the component level, so that is why I gave it a default value.